sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/implementation/TextAnalyticsClientImpl.java

```diff
@@ -48,7 +49,7 @@
 
     /**
      * Gets Supported Cognitive Services endpoints (protocol and hostname, for example: https://westus.api.cognitive.microsoft.com).
-     *
+     * 
```

Comment:
Seems like this random whitespace change has made it's way into the PR, can you remove these?

Reply:
Will keep as it is since it is auto-generated code.

Reply:
You can skip review all files that with this license comment at the top of file
```
// Copyright (c) Microsoft Corporation. All rights reserved.
// Licensed under the MIT License.
// Code generated by Microsoft (R) AutoRest Code Generator.
```